ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -479,12 +484,51 @@ class PackedBubbleSeries extends BubbleSeries {
         seriesProto.destroy.apply(this, arguments as any);
     }
 
+    /**
+     * PackedBubble needs a custom initDataLabels function
+     * to initiate the animation in a different way
+     * than for other series (drawDataLabels is called each frame)
+     * @private
+     */
+    public initDataLabels(this: PackedBubbleSeries): SVGElement {
```

Comment:
As far as I can see this implementation is identical to the one in NetworkgraphSeries.ts, which means we're violation the [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Sooner or later someone will fix something in one place, not knowing about the other. So we should put this function in a module and import it into both series types. I am not sure if we have a fitting module for sharing stuff between packed bubble and network graph, otherwise we can create a new one. @pawelfus please advice.

Reply:
We have files that are being shared between those series (layout-related ones), but `initDataLabels` doesn't suit there. I think a simple composition similar to `DragNodesComposition.ts` or `GraphLayoutComposition.ts` should be enough. Unless we can use something that fits better (like 2010' mixins)?

Reply:
Or something similar to `DrawPointUtilities.ts`? 🤔

Reply:
Yes, I don't think we need to make a composition out of it, just some module that shares that function. It can even export _only_ that function (as `export default`). In the future, if we identify more shared needs, we can just refactor it.